### ISSUE TITLE
Mappable variants are missing information from the mapping pipeline, though they are found in Ensembl

### ISSUE THREAD

**Issue by ljwh2** [EBISPOT/goci#1045]
In some cases, mappable variants do not return any information e.g. CHR_ID and CHR_POS, MAPPED_GENE, even though this information is visible in Ensembl and sometimes in the LD plot. This needs investigating and fixing. 
Two types of example:
1. The first are missing CHR_ID, CHR_POS, MAPPED GENE, GENE_ID, and no LD plot is displayed. Does display alleles, minor allele and MAF (visible in info box of UI). CONTEXT is displayed in the associations download (but this is missing in the UI - displayed as “Most severe consequence”). 
https://www.ebi.ac.uk/gwas/variants/rs1486471
https://www.ebi.ac.uk/gwas/variants/rs7833268 
2. Same as 1, except the LD plot _is_ displayed correctly
https://www.ebi.ac.uk/gwas/variants/rs1919484
https://www.ebi.ac.uk/gwas/variants/rs11385942 

For comparison, here are some examples where mapping info is missing but correct:
3. No mapping, variant does not exist in Ensembl: https://www.ebi.ac.uk/gwas/variants/rs73070466
4. Location mapping but no gene, this is a SNP that does not fall in or near a gene: https://www.ebi.ac.uk/gwas/variants/rs2144366

Of note, Daniel has found that some variants are inconsistently mapped between releases. See [here](https://docs.google.com/spreadsheets/d/12AUMzjcNzalbYuArS3EtL0aJwl_J-rynQU7KPyg4vpw/edit?usp=sharing) for a matrix of ~500 SNPs which had inconsistent mapping (CHR_ID, CHR_POS present or absent) for a random sample of 5 releases between 2019 and 2023.  He says:
"There's no obvious pattern in the affected variants across releases. There might be a weak correlation with the number of associations indicating a stochastic effect in the background. Some release seems more luckier than others." More details are here: https://gist.github.com/DSuveges/95d1db92bec67fcb1cfba0df8053f94c

**Comment by ljwh2** [comment]
Daniel has found that some variants are inconsistently mapped between releases. See attached file for a matrix of ~500 SNPs which had inconsistent mapping (CHR_ID, CHR_POS present or absent) for a random sample of 5 releases between 2019 and 2023.  He says:
"There's no obvious pattern in the affected variants across releases. There might be a weak correlation with the number of associations indicating a stochastic effect in the background. Some release seems more luckier than others." More details are here: https://gist.github.com/DSuveges/95d1db92bec67fcb1cfba0df8053f94c

**Comment by ljwh2** [comment]
![image.png](https://images.zenhubusercontent.com/5e57ecb5d0668b3a20e94b9e/45b66c3f-6504-487e-8559-5f0ba2558524)

**Comment by ljwh2** [comment]
@sajo-ebi to clarify, the goal for this sprint is:
1. get a good understanding of the mapping pipeline and how it works
2. Find out where in the pipeline the two main types of failure occurred i.e. what does this “mapping failure” mean from the technical point of view?
3. Create a technical ticket for next step.

**Comment by ljwh2** [comment]
Attached is a list of SNPs without mapping. To define this list I pulled out: 
1. any single SNP with resolvable rsID i.e. in format rs123456  
2. Any of 1 without any genomic location (defined as numeric value in in CHR_ID field)
Then
3. Checked these against Ensembl using the SNP validation tool in DepoCuration  (result in Column B) 
4. Checked these against a list of multi-SNPs (result in column C)
5. Checked a few to see whether the LD plot was displayed (result in column D)

Considering (4), around 15% of unmapped SNPs are found in a multi-SNP, whereas these only account for 5% of total SNPs in the Catalog, related issue goci#1046 may have an impact for those. 

[https://app.zenhub.com/files/2995118/5ba90bd2-af73-4e55-bbb3-790c08815376/download](https://app.zenhub.com/files/2995118/5ba90bd2-af73-4e55-bbb3-790c08815376/download)
